std/math.d

```diff
@@ -8314,27 +8325,89 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     return impl(x, y);
 }
 
+/*
+Of course, floating point numbers should be compared using isClose,
+but the main purpose of this test is to provide a good example,
+not to test the pow function, and this would lead to ugly examples like:
```

Comment:
A good example is one that is appropriate to imitate, not just look at. Users should not be comparing the results of this `pow` function using `==`, so the examples shouldn't either.

As noted in the author's **TODO**, the `exp2(y * log2(x))` method currently used is very imprecise in some parts of its domain, so this isn't just quibbling about the occasional single-bit rounding error. Using `isClose`, `feqrel`, or `approxEqual` serves the practical purpose of warning the user about this, in addition to enabling automatic verification of the example.